
public class problem {
public static void main(String[]args) {
	int b;
try {	
	b=10/0;}
catch(Exception e ){
	
	System.out.println("exception handled");
}
	System.out.println("ourside");
}
}
